pkg/apis/experimental/v1/types.go

```diff
@@ -541,3 +541,64 @@ type IngressBackend struct {
 	// Specifies the protocol of the referenced service.
 	Protocol v1.Protocol `json:"protocol,omitempty"`
 }
+
+// DerivedNodeMetricsList contains derived metrics for a single node. It holds
+// derived metrics in windows about the node itself and all of its system
+// containers.
+type DerivedNodeMetrics struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Standard list metadata.
+	v1.ObjectMeta `json:"metadata,omitempty"`
+
+	// NodeMetrics contains derived metrics about the node.
+	NodeMetrics MetricsWindows `json:"nodeMetrics"`
+	// SystemContainers contains derived container metrics for all system
+	// containers in the node.
+	SystemContainers []DerivedContainerMetrics `json:"systemContainers"`
+}
+
+// DerivedNodeMetricsList contains derived metrics for a single container.
+type DerivedContainerMetrics struct {
+	// ContainerName is the name of the container.
```

Comment:
Use json field names, not Go field names in all comments on API fields.

Also, what do you mean by "name of the container"? Could you be more specific? Does this relate to any fields in the Pod API? Is it what we call containerID?
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L1088